run/markdown-preview/noxfile.py

```diff
@@ -0,0 +1,186 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import os
+from pathlib import Path
+
+import nox
+
+
+# DO NOT EDIT - automatically generated.
+# All versions used to tested samples.
+ALL_VERSIONS = ["2.7", "3.6", "3.7", "3.8"]
+
+# Any default versions that should be ignored.
+IGNORED_VERSIONS = ["2.7", "3.6", "3.8"]
```

Comment:
The container is running 3.8-slim, so we probably only want to test on 3.8?
```suggestion
IGNORED_VERSIONS = ["2.7", "3.6", "3.7"]
```

Reply:
We're actually hijacking the 3.7 tests, and running the cloud_run session instead.  If we update here, we would have this in the py() function as well:
```
if session.python == '3.7':
        cloud_run(session)
```

Reply:
Oh ok - do we intended to commit using this or can we add the Cloud Run check to Kokoro before?

Reply:
What do you think is easier to maintain? @tmatsuo for his opinion as well.

Reply:
I'm ok with either way, with a slight preference of merging this as it is.

Reply:
Ok - SGTM.